lib/puppet/util.rb

```diff
@@ -323,7 +323,7 @@ def path_to_uri(path)
     params = { :scheme => 'file' }
 
     if Puppet.features.microsoft_windows?
-      path = path.gsub(/\\/, '/')
+      path = path.tr('\\', '/')
```

Comment:
You could use `path.tr!`

Reply:
Since `path` is passed in, I don't want to mutate the callers variable.